stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/info/group/users/GroupChatInfoAddUsersViewModel.kt

```diff
@@ -4,24 +4,32 @@ import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.asLiveData
 import androidx.lifecycle.viewModelScope
 import io.getstream.chat.android.client.ChatClient
 import io.getstream.chat.android.client.api.models.QueryUsersRequest
 import io.getstream.chat.android.client.call.await
 import io.getstream.chat.android.client.models.Filters
 import io.getstream.chat.android.client.models.Member
 import io.getstream.chat.android.client.models.User
-import io.getstream.chat.android.livedata.ChatDomain
-import io.getstream.chat.android.livedata.controller.ChannelController
+import io.getstream.chat.android.core.internal.InternalStreamChatApi
 import io.getstream.chat.android.livedata.utils.Event
+import io.getstream.chat.android.offline.experimental.channel.state.ChannelState
+import io.getstream.chat.android.offline.experimental.extensions.asReferenced
 import kotlinx.coroutines.launch
 
 class GroupChatInfoAddUsersViewModel(
     cid: String,
-    chatDomain: ChatDomain = ChatDomain.instance(),
     chatClient: ChatClient = ChatClient.instance(),
 ) : ViewModel() {
 
+    /**
+     * Holds information about the current channel and is actively updated.
+     */
+    @OptIn(InternalStreamChatApi::class)
```

Comment:
It's because `ChannelState` and `QueryChannelState` are annotated with `@InternalStreamChatAPI`, right? We should remove this annotation and those opt ins.

Reply:
Yup, `ChannelState` is marked as internal.

I can remove the annotations in a clean up PR if you want.